nodebuilder/node/flags.go

```diff
@@ -0,0 +1,79 @@
+package node
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+)
+
+const (
+	// DefaultNetwork is the default network of the current build.
+	DefaultNetwork   = Arabica
+	EnvCustomNetwork = "CELESTIA_CUSTOM"
+	nodeNetworkFlag  = "node.network"
+)
+
+// Flags gives the set of flags that define the network parameters
+func Flags() *flag.FlagSet {
+	flags := &flag.FlagSet{}
+
+	flags.String(
+		nodeNetworkFlag,
+		"",
+		"The name of the network to connect to, e.g. "+listProvidedNetworks(),
+	)
+
+	return flags
+}
+
+// ParseFlags tries to parse the network from the flags and environment,
+// and returns either the parsed network or the build's default network
+func ParseFlags(cmd *cobra.Command) (Network, error) {
+	parsedNetwork := cmd.Flag(nodeNetworkFlag).Value.String()
+	// no network set through the flags, so check if there is an override in the env
+	if parsedNetwork == "" {
+		return ParseNetworkFromEnv()
+	}
+	return Network(parsedNetwork), nil
+}
+
+// ParseNetworkFromEnv tries to parse the network from the environment,
+// and returns either the parsed network or the build's default network
+func ParseNetworkFromEnv() (Network, error) {
+	var network Network
+	// check if custom network option set
+	// format: CELESTIA_CUSTOM=<netID>:<genesisHash>:<bootstrapPeerList>
+	if custom, ok := os.LookupEnv(EnvCustomNetwork); ok {
```

Comment:
Looks like it returns empty network if there is no env variable, but comment suggests there should be default one. Maybe there is some guarantee that env variable will persist?